spec/meilisearch/index/search/matching_strategy_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
```

Comment:
I don't understand this comment :) Why is it here and what is it?

Reply:
I found something on Stackoverflow: https://stackoverflow.com/questions/37799296/ruby-what-does-the-comment-frozen-string-literal-true-do

My tutor during my first internship also told me it was more performant. Maybe @brunoocasali will be able to say if it's true or not haha

Reply:
Yeaah, that's correct with this magic comment. You will save memory because you don't need to allocate more space to store the same string.

In Ruby 3.0. Matz (Ruby’s creator) decided to make all String literals frozen by default.